Lesson2/Holodenko_Artem/task_1/main.cpp

```diff
@@ -0,0 +1,74 @@
+#include <iostream>
+#include <stack>
+#include <string>
+
+void print_stack(std::stack<char> stack)
+{
+    while(!stack.empty())
+    {
+        std::cout << stack.top() << " ";
+        stack.pop();
+    }
+    std::cout << std::endl;
+}
+
+std::stack<char> string_to_stack(std::string *_str)
+{
+    std::stack<char> data_stack;
+    const char* str = _str->c_str();
```

Comment:
**wtf?**

Reply:
1. You pass string as pointer, and don't check if it is null
2. You're manipulating C string directly. WHY!?
You don't use iterators at all.
3. You're doing weird things